app/controllers/api/v1/downloads_controller.rb

```diff
@@ -44,6 +44,26 @@ def batch_export
         download(storage, file_location)
       end
 
+      # GET /downloads/batch_import/:id(/:option)
+      def batch_import
+        batch_import = load_and_authorize_batch_import!
+
+        if batch_import_params[:option] == 'without_successful_imports'
+          send_data(
+            batch_import.csv_without_successful_imports,
+            type: 'text/csv',
+            filename: "without-successful-rows-#{batch_import.original_filename}"
```

Comment:
We call it "without-successful-rows" in Hyacinth 2, but I don't mind if we change it to "without-successful-imports" in Hyacinth 3 if you don't.

Reply:
Are you talking about the filename prefix?

Reply:
Yeah exactly.  We could do: `"without-successful-imports-#{batch_import.original_filename}"`

Reply:
ahhh! I have no strong feelings. Would you prefer "without-successful-imports-#{batch_import.original_filename}"?

Reply:
I have a slight preference for it just because of the future scenario of creating batch imports that aren't from CSVs (like from voyager-to-hyacinth syncs).

Reply:
Okay, I'll make the change.

Reply:
Thanks!